Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPLauncherFolder)'==''" Text="Please specify UAPLauncherFolder" />
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
```

Comment:
No. We have two different properties. `BuildingUAPVertical` and `BuildingUAPAOTVertical`